src/app/about/page.tsx

```diff
@@ -8,50 +10,63 @@ export default function About() {
 
       <div className="m-auto max-w-5xl w-full mt-12">
         <h1 className="text-2xl">
-          Om projektet
+          About the project
         </h1>
 
         <p className="mt-2">
-          Projektet är skapat av   <a
+          This project is created by<a
             className="lg:pointer-events-auto pointer-events-none italic hover:underline"
             href="https://www.johanehrenfors.se"
             target="_blank"
             rel="author"
           >
-            Johan Ehrenfors&nbsp;  
+            Johan Ehrenfors&nbsp;
           </a>
-         med bidrag från&nbsp;
+          with contributions from&nbsp;
           <a
             className="lg:pointer-events-auto pointer-events-none italic hover:underline"
             href="https://www.fullstackfrancis.com"
             target="_blank"
             rel="author"
           >
-             Francis Jones
-          </a>, i Typescript och Next.js.
-          
+            Francis Jones
+          </a>.
+
         </p>
 
         <p className="mt-2">
-          Som många andra, så har vi glömt att flytta bilen inför en städdag, vilket har resulterat i saftiga böter.
-          Vi ville snabbt kunna ta reda på vart vi kan låta bilen stå så länge som möjligt, men även ett smidigt sätt att få relevanta påminnelser när det är dags att flytta bilen igen.
+          Like many others, we&apos;ve forgotten to move our car before a cleaning day, resulting in hefty fines.
+          We wanted to quickly find out where we could leave our car for as long as possible, as well as a convenient way to receive relevant reminders when it&apos;s time to move our car again.
         </p>
 
+        {/* notifikationer kommer bli klara när vi lanserad detta   <p className="mt-2">
```

Comment:
Still in Swedish

Reply:
I think we can get rid of this paragraph, thoughts?

Reply:
Sure!